conjure_oxide/tests/integration/basic/min/01/input.expected-minion.solutions.json

```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "2": 3,
```

Comment:
I suspect "2" here is the machine name's number? why not "1"?

Reply:
As a slight optimization I start at the length of the current symbol table and count upward until I find a variable name which isn't present. This avoids counting through a lot of lower values every time.

Reply:
oh I think we should remember a single int (in the symbol table?) through which we know what the next machine name should be called. like good old gensym of lisp. have a look if you haven't come across this before.

Reply:
That makes sense! Will do.